modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-consume.adoc

```diff
@@ -0,0 +1,92 @@
+= Anypoint MQ Consume Operation
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The `consume` operation in the Anypoint MQ connector provides the ability to consume a single message at any time of the flow, from any destination, using the acknowledgment strategy that supports your use case.
+
+== Incoming Message Structure
+
+Every consumed message consists of two parts:
```

Comment:
The title identifies the message as "incoming" but the first sentence as "consumed": which is the more precise? Consider standardizing to avoid confusion.